pulsar-broker/src/test/java/org/apache/pulsar/broker/service/BrokerServiceAutoTopicCreationTest.java

```diff
@@ -104,6 +104,24 @@ public void testAutoTopicCreationDisableIfNonPartitionedTopicAlreadyExist() thro
         assertTrue(admin.namespaces().getTopics("prop/ns-abc").contains(topicName));
     }
 
+    @Test
+    public void testAutoSubscriptionCreationDisable() throws Exception{
```

Comment:
@murong00 

Can you add a test to cover the case - if `allowAutoSubscriptionCreation` is disabled people can still create a subscription using `pulsar-admin`?

Reply:
Added.